app/services/solr_mapper.rb

```diff
@@ -13,24 +13,26 @@ def self.call(...)
 
   # @param metadata [Hash] the Dataworks metadata
   # @param doi [String] the DOI, if present, stored in the dataset record
-  def initialize(metadata:, doi:, dataset_record_id:, dataset_record_set_id:)
+  # @param id [String] the ID of the dataset
+  # @param load_id [String] the ID of the load
+  def initialize(metadata:, doi:, id:, load_id:)
     @metadata = metadata.with_indifferent_access
     @doi = doi
-    @dataset_record_id = dataset_record_id
-    @dataset_record_set_id = dataset_record_set_id
+    @id = id
+    @load_id = load_id
   end
 
   # @return [Hash] the Solr document
   # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
   def call
     {
-      id: dataset_record_id,
-      dataset_record_set_id_ss: dataset_record_set_id,
+      id:,
+      load_id_ss: load_id,
       access_ssi: metadata['access'],
       provider_ssi: metadata['provider'],
       descriptions_tsim: descriptions_field,
       doi_ssi: doi,
-      provider_identifier_ssi: provider_identifier_field,
```

Comment:
This should still end with the suffix "_ssi". thanks!

Reply:
I will be on the look out for that.  That is the field in the staging Solr, but will check what happens when I run this locally.